Showcase-iOSTests/Login/Presenter/LoginPresenterTests.swift

```diff
@@ -120,6 +131,11 @@ class LoginPresenterTests: XCTestCase {
                 userDefaultsDictionary["\(key)"] = value
             }))
         }
+        
+        stub(mockLoginViewer) { (mock) in
+            _ = when(mock.stopLoadingAnimation().thenDoNothing())
```

Comment:
you should be verifying that these get called please, everywhere that you stubbed. here and above